website/docs/components/side-nav/partials/guidelines/guidelines.md

```diff
@@ -108,9 +108,9 @@ Use `isLinkExternal` to show that the list item is a hyperlink pointing to a pag
 Use external links sparingly. Avoid using this property to link pages that are unrelated to the product's navigation.
 !!!
 
-### Custom content
+### Generic content
 
-Toggle `hasCustomContent` on to support any additional custom content, local components, or Helios components within the body container via an instance swap property (customInstance) in Figma.
+Toggle `hasGenericContent` on to support any additional generic content, local components, or Helios components within the body container via an instance swap property (genericInstance) in Figma.
```

Comment:
@jorytindall @andgen404 These may have been reverted when you merged up your branch. They still say `hasTopCustomContent`, `topCustomContent`, `hasBottomCustomContent`, and `bottomCustomContent` in Figma.

```suggestion
The topGenericInstance and bottomGenericInstance properties support any additional generic content, local components, or Helios components within the body container via instance swap properties (`topGenericInstance`, `bottomGenericInstance`) in Figma.
```

Reply:
Ah, yeah. I made note of this. Thank you! @heatherlarsen